util/generate-sample-ledger/src/lib.rs

```diff
@@ -70,9 +70,10 @@ pub fn bootstrap_ledger(
     log::info!(logger, "recipients: {}", recipients.len());
     log::info!(
         logger,
-        "Making {:?} outputs of {:?} picoMOB.",
+        "Making {:?} outputs of {:?} picoMOB, up to token ID {:?}.",
         num_outputs,
-        picomob_per_output
+        picomob_per_output,
+        max_token_id,
```

Comment:
so, there's an issue here which is, we added `max_token_id` thinking it would work, but in fact it doesn't work. I'm not sure exactly why but I've never been able to get alternate tokens that are spendable by fog distro.

I think we should actually just delete the `--max-token-id` thing, we've already founder other solutions where the custom tokens are minted normally